server/data/hmppsManageUsersClient.test.ts

```diff
@@ -0,0 +1,59 @@
+import nock from 'nock'
+
+import HmppsManageUsersClient from './hmppsManageUsersClient'
+import config from '../config'
+import type { User } from '@accredited-programmes/users'
+
+jest.mock('./tokenStore')
+
+const token = 'token-1'
+
+describe('HmppsManageUsersClient', () => {
+  let fakeManangeUsersApiUrl: nock.Scope
+  let hmppsManageUsersClient: HmppsManageUsersClient
+
+  beforeEach(() => {
+    fakeManangeUsersApiUrl = nock(config.apis.hmppsManageUsers.url)
+    hmppsManageUsersClient = new HmppsManageUsersClient(token)
+  })
+
+  afterEach(() => {
+    if (!nock.isDone()) {
+      nock.cleanAll()
+      throw new Error('Not all nock interceptors were used!')
+    }
+    nock.abortPendingRequests()
+    nock.cleanAll()
+  })
+
+  describe('getCurrentUsername', () => {
+    it('should return data from api', async () => {
+      const response: Pick<User, 'username'> = { username: 'DEL_HATTON' }
+
+      fakeManangeUsersApiUrl.get('/users/me').matchHeader('authorization', `Bearer ${token}`).reply(200, response)
+
+      const output = await hmppsManageUsersClient.getCurrentUsername()
+      expect(output).toEqual(response)
+    })
+  })
+
+  describe('getUserFromUsername', () => {
+    it('should return data from api', async () => {
```

Comment:
I know it's kind of implicit here, but how about: "should return the user from the API"?